src/instructlab/generator/generate_data.py

```diff
@@ -81,14 +111,21 @@
 ]
 
 
-def check_prompt_file(prompt_file_path):
+def check_prompt_file(prompt_file_path, model_family):
     """Check for prompt file."""
     try:
         with open(prompt_file_path, encoding="utf=8") as file:
             prompt_template = file.read()
     except FileNotFoundError:
-        print(f"Cannot find {prompt_file_path}. Using default prompt.")
-        prompt_template = DEFAULT_PROMPT_TEMPLATE
+        print(
+            f"Cannot find {prompt_file_path}. Using default prompt depending on model-family."
+        )
```

Comment:
Any reason we are using `print` here instead of the logger?

Reply:
This was always a print, I just changed what was being printed. Opting to not change logic in this PR if I don't have to.

Reply:
fair enough!